src/main/utils/FactApi.ts

```diff
@@ -109,4 +109,17 @@ export class FactApi {
         };
       });
   }
+
+  public courtTypesSearch(courtTypes: string): Promise<Array<CourtDetailsResult>> {
+    return this.axios
+      .get(`courts/court-types/${courtTypes}`)
+      .then(results => results.data)
+      .catch(err => {
+        this.logger.error(err);
+        return {
+          courts: []
```

Comment:
Returning a 500 response or an error page may be better? But if so prob one for another card? Thoughts? 

Just thinking if the user sees no results it may indicate the API is down or so